presidio-anonymizer/tests/anonymizers/test_validators.py

```diff
@@ -34,22 +57,13 @@ def test_when_actual_parameter_is_non_json_typed_then_ipe_raised_with_general_er
 
 def test_given_wrong_type_then_we_fail():
     err_str = "Invalid parameter value for name. Expecting 'string', but got 'number'."
-    with pytest.raises(
-            InvalidParamException,
-            match=err_str
-    ):
-        validate_type(
-            parameter_value=1, parameter_name="name", parameter_type=str
-        )
+    with pytest.raises(InvalidParamException, match=err_str):
+        validate_type(parameter_value=1, parameter_name="name", parameter_type=str)
```

Comment:
black autoformatted it for me, so in case we'll change this file again it will make the format as it was again. We can have discussion on using black again, or try to refine it somehow since it gives some unreadable results sometimes

Reply:
so what's to do here?

Reply:
What did the reformatting for you? IDE?